src/main/java/com/marklogic/batch/rdbms/ColumnMapUriGenerator.java

```diff
@@ -0,0 +1,20 @@
+package com.marklogic.batch.rdbms;
```

Comment:
Should this in MSB instead? Seems like pretty simple plumbing, and I'd like to minimize the amount of Java code in this starter project so that there are as few moving parts as possible for getting started.